widget/embedded/src/containers/QuoteInfo/QuoteInfo.tsx

```diff
@@ -19,25 +31,28 @@ export function QuoteInfo(props: PropTypes) {
     (error.type === QuoteErrorType.NO_RESULT ||
       error.type === QuoteErrorType.REQUEST_FAILED);
 
-  const showQuote = !noResult && quote && quote.result && !loading;
+  const showQuote = !noResult && quote && !loading;
 
   if (loading) {
     return (
       <QuoteContainer>
-        <QuoteSkeleton type={type} expanded={expanded} />
+        <QuoteSkeleton tag={!tagHidden} type={type} expanded={expanded} />
       </QuoteContainer>
     );
   }
 
   return showQuote ? (
-    <QuoteContainer>
+    <QuoteContainer onClick={() => onClick && onClick(quote)}>
```

Comment:
You can simplify it:
onClick={() => onClick?.()}

Reply:
fixed